src/config.h

```diff
@@ -134,6 +138,8 @@ class Config {
   ImportConfig import;
 
  private:
+  std::vector<boost::filesystem::path> config_dirs_ = {"/usr/sota/conf.d/", "/etc/sota/conf.d/"};
```

Comment:
Should this be `/usr/lib/sota/conf.d` to match the current directory to look for configs?